include/crocoddyl/core/solvers/ipopt/ipopt.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef CROCODDYL_CORE_SOLVERS_IPOPT_IPOPT_HPP_
+#define CROCODDYL_CORE_SOLVERS_IPOPT_IPOPT_HPP_
+
+#include "coin-or/IpIpoptApplication.hpp"
+
+#include "crocoddyl/core/solver-base.hpp"
+#include "crocoddyl/core/solvers/ipopt/multiple-shooting-nlp.hpp"
```

Comment:
Renamed this file to ipopt-iface.hpp, i.e.,
```suggestion
#include "crocoddyl/core/solvers/ipopt/ipopt-iface.hpp"
```